tests/Types.hx

```diff
@@ -187,4 +187,16 @@ enum Content {
 typedef OptionalFinal = {
   @:optional final i:Int;
 }
-#end
\ No newline at end of file
+#end
+
+
+
+enum AnyInlineObject {
+	@:json('') Text;
```

Comment:
Should `Text` not have a parameter to contain the actual text?

Reply:
Well since this is an enum, it is supposed to match the exact string value.
But now I think your @:jsonParse solution in gitter is more generic